docs/SUMMARY.md

```diff
@@ -82,6 +82,7 @@
   * [.feastignore](reference/feature-repository/feast-ignore.md)
 * [Feature servers](reference/feature-servers/README.md)
   * [Local feature server](reference/feature-servers/local-feature-server.md)
+  * [Go feature server](reference/feature-servers/go-feature-server.md)
```

Comment:
Everything else looks good, but the only nit I have is that Go feature server is a bit misleading since it's not just Go, and there's no standalone go feature server that can be deployed.

I'd probably called this Go-based feature retrieval and update the docs and flag. What do you think?

Reply:
I left it in the `feature servers` folder since I think out of all folders it's still most relevant to Feature Servers.